<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyApp</class>
 <widget class="QDialog" name="MyApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчеты ВЭС</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>161</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>Типы отчетов</string>
   </property>
   <widget class="QCheckBox" name="checkVzlet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Взлет</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkVKT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ВКТ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkMKTS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>70</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>МКТС</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSPT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>70</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>СПТ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkTV7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>70</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ТВ-7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkTMK">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ТМК</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить файлы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Укажите путь </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>261</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать обработку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>  Путь до сохранения не выбран</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>241</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчистить список файлов</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>31</y>
     <width>241</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбранные файлы</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Обработанные файлы</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
